pkg/helpers/helpers.go

```
@@ -0,0 +1,10 @@
+package helpers
+
+// Helper function to create pointers to int64 and float64 values.
```

golangci-lint

[error] ST1020: comment on exported function Int64Ptr should be of the form "Int64Ptr ..."

(stylecheck)